webpack/index.js

```diff
@@ -0,0 +1,22 @@
+import URI from 'urijs';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { mount } from 'foremanReact/common/MountingService';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import componentRegistry from 'foremanReact/components/componentRegistry';
+import JobInvocationContainer from './react_app/components/job_invocations';
+import './react_app/redux/reducers';
```

Comment:
Prefer to import variables and use them instead importing code that supposes to run and work.
Can migrate to:

`webpack/react_app/redux/reducers/index.js`
```jsx
export default combineReducers({
  jobInvocations,
});
```

`webpack/index.js`
```jsx
import { registerReducer } from 'foremanReact/common/MountingService';
import rootReducer from './react_app/redux/reducers';

registerReducer('foreman_remote_execution_reducers', rootReducer);
```